scripts/rebranding.ps1

```diff
@@ -0,0 +1,109 @@
+#     following disclaimer.
+# *   Redistributions in binary form must reproduce the above
+#     copyright notice, this list of conditions and the
+#     following disclaimer in the documentation and/or other
+#     materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+# CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+# INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+# SUCH DAMAGE.
+
+Param(
+    [Parameter(HelpMessage = "Global build number")]
+    [int]$buildNumber
+)
+
+function version_csharp([string]$file) {
+    Write-Host "Versioning file $file"
+
+    (Get-Content -Path $file -Encoding "utf8") `
+        -replace "0.0.0.0", "$BRANDING_XC_PRODUCT_VERSION.$buildNumber" `
+        -replace "0000", "$BRANDING_XC_PRODUCT_VERSION.$buildNumber" |`
+        Set-Content -Path $file -Encoding "utf8"
+}
+
+function rebranding_global([string]$file) {
+    Write-Host "Rebranding file $file"
+
+    (Get-Content -Path $file -Encoding "utf8") `
+        -replace "\[Vendor Legal\]", $BRANDING_COMPANY_NAME_LEGAL `
+        -replace "\[Vendor\]", $BRANDING_COMPANY_NAME_SHORT `
+        -replace "\[Guest Tools\]", $BRANDING_PV_TOOLS `
+        -replace "\[XenServerProduct\]", $BRANDING_PRODUCT_BRAND `
+        -replace "\[XenServer version\]", $BRANDING_PRODUCT_VERSION_TEXT `
+        -replace "\[XenServer host\]", $BRANDING_SERVER `
+        -replace "\[XenCenter\]", $BRANDING_BRAND_CONSOLE `
+        -replace "\[XenCenter_No_Space\]", $BRANDING_BRAND_CONSOLE_NO_SPACE `
+        -replace "xencenter/current-release/", $BRANDING_HELP_PATH `
+        -replace "\[Xc updates url\]", $XC_UPDATES_URL `
+        -replace "\[Cfu url\]", $CFU_URL `
+        -replace "\[YumRepoBaseBin\]", $YUM_REPO_BASE_BIN `
+        -replace "\[YumRepoBaseSource\]", $YUM_REPO_BASE_SRC `
+        -replace "\[YumRepoEarlyAccessBin\]", $YUM_REPO_EARLY_ACCESS_BIN `
+        -replace "\[YumRepoEarlyAccessSource\]", $YUM_REPO_EARLY_ACCESS_SRC `
+        -replace "\[YumRepoNormalBin\]", $YUM_REPO_NORMAL_BIN `
+        -replace "\[YumRepoNormalSource\]", $YUM_REPO_NORMAL_SRC |`
+         Set-Content -Path $file -Encoding "utf8"
+}
+
+Write-Host "Started product rebranding"
+
+$REPO = Get-Item "$PSScriptRoot\.." | Select-Object -ExpandProperty FullName
+
+. $REPO\scripts\branding.ps1
```

Comment:
White space indent

Reply:
It's not whitespace, it's dot space filepath, used to include a different script in the current one.

Reply:
ah did not know that, then no worries